deploy/00_Deploy_Mocks.js

```diff
@@ -1,27 +1,38 @@
-const DECIMALS = '18'
-const INITIAL_PRICE = '200000000000000000000'
-
 module.exports = async ({
     getNamedAccounts,
     deployments,
     getChainId
 }) => {
+    const DECIMALS = '18'
+    const INITIAL_PRICE = '200000000000000000000'
     const { deploy, log } = deployments
     const { deployer } = await getNamedAccounts()
-    let chainId = await getChainId()
-    let ethUsdPriceFeedAddress
+    const chainId = await getChainId()
     // If we are on a local development network, we need to deploy mocks!
     if (chainId == 31337) {
-        log("Local network deteteched! Deploying mocks...")
-        linkToken = await deploy('LinkToken', { from: deployer })
-        await deploy('MockV3Aggregator', { from: deployer, args: [DECIMALS, INITIAL_PRICE] })
-        await deploy('VRFCoordinatorMock', { from: deployer, args: [linkToken.address] })
-        await deploy('MockOracle', { from: deployer, args: [linkToken.address] })
+        log("Local network detected! Deploying mocks...")
+        const linkToken = await deploy('LinkToken', { from: deployer, log: true })
+        await deploy('EthUsdAggregator', {
```

Comment:
Is this confusing? We then refer it back to mockaggregator later?

Reply:
This is to future-proof the deployment. When you can have multiple uses of the same contract, it's best to give them a unique name. In this case `EthUsdAggregator` is specific to the data used by the contract. That way if we wanted to later add a mock for `BtcUsdAggregator` is simple to add it in and use that in a test.